### PR TITLE
[FEATURE] crosschain-swap-api integration - `getQuotes` endpoint

### PR DESCRIPTION
**Pull request by yuripramos**
## Are you opening a new PR? Please fill the checklist below

[] Does the code work? Check whether function and logic are correct.
[x] Are functions, methods, and variables adequately named?
[x] Are names semantic and meaningful for your business?
[x] Is it manually tested?
[x] Is the code clear and easy to understand?
[] Is there any commented or unnecessary code? Try to avoid as much as possible, but if you let comments, explain WHY in the PR
[] Documentation needs to be updated?
[x] Is the code as modular as possible?
[] Does the code consider failures? Is it just considering the happy path?
[] I have tested not only the happy path, but possible roots that can lead to code regression
[] Is there any performance issue?

This PR is related only to **getQuote** endpoint with a few considerations:

1.  This solution has some temporary workarounds
2.  Usage of `temp/nxtp-yuri` branch instead of `main` to hit `crosschain-swap-api-web-frontend.dev.swing.xyz` endpoint to get specific nxtp response to make able to proceed further steps in the frontend without using `/transfer` and `/claim` endpoints yet
4. FIltering only cbridge and nxtp as supported networks during the process of migration because our frontend currently only supports these bridges so far. i.e. until we migrate the whole application I'm displaying only cbridge and nxtp as possible routes.

### CODE REVIEW

File: src/client/js/utils/wallet.js [hunk 1]
```diff
@@ -300,6 +300,31 @@ window.WalletJS = {
     await TokenListManager.updateTokenList();
   },
 
+  returnEstimatedReturnAmountDeductedByFees: function (estimateTx) {
```
Comment (chillvybz) [inline src/client/js/utils/wallet.js:303]: Tthis seems like useful logic to exist in the API itself.
cc @8d526

File: src/client/js/utils/txBridgeManager.js [hunk 2]
```diff
@@ -53,169 +27,114 @@ export default {
...
     const tx = this.getTx(nonce);
+
     if (!tx) {
       return false;
     }
 
-    return tx.bridge === 'connext' || tx.bridge === 'cbridge';
+    const bridgeName = tx.bridge?.route[0]?.bridge;
+
+    return bridgeName === 'nxtp' || bridgeName === 'celer';
```
Comment (chillvybz) [inline src/client/js/utils/txBridgeManager.js:130]: this should from from the API

File: src/client/js/utils/txBridgeManager.js [hunk 3]
```diff
@@ -226,8 +145,6 @@ export default {
   },
 
   buildGenericTxHistory(bridgeTxHistory, bridge) {
-    // TODO: Implement Hop TxHistory
```
Comment (chillvybz) [inline src/client/js/utils/txBridgeManager.js:229]: is history integratoim from the API coming later?

File: src/client/js/utils/cbridge.js [hunk 4]
```diff
@@ -13,10 +13,7 @@ import Wallet from './wallet';
 import TokenListManager from './tokenList';
 import { abi as BridgeABI } from '../../../contract/celer_contract/Bridge.json';
```
Comment (chillvybz) [inline src/client/js/utils/cbridge.js:14]: are these cbridge.js and nxtp.js files still relevant ? if it's abstracted away by the API?

File: src/client/js/utils/requests/utils.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+// 'https://api.dev.swing.xyz'
+
+// Temporary baseUrl until merge /transfer and /claim
+const baseUrl = 'https://crosschain-swap-api-web-frontend.dev.swing.xyz';
```
Comment (chillvybz) [inline src/client/js/utils/requests/utils.js:4]: should be an injected ENV_VAR from webpack

File: src/client/js/components/partials/bridge/BridgeWidget.jsx [hunk 6]
```diff
@@ -27,17 +27,6 @@ export default class BridgeWidget extends Component {
       (v) => v.enabled && v.crossChainSupported,
     );
 
-    /*
-    // Move ETH to last, it has too high of fees
-    this.CROSS_CHAIN_NETWORKS.push(
```
Comment (chillvybz) [inline src/client/js/components/partials/bridge/BridgeWidget.jsx:32]: why you delete?

**Review by chillvybz — CHANGES_REQUESTED** [review]

**Comment by chillvybz** [comment]
I re-read your PR comment and answers some of my questions but curious to hear more of your thoughts about some of the particular items.

File: src/client/js/utils/txBridgeManager.js (hunk 2 above)
Comment (yuripramos) [reply src/client/js/utils/txBridgeManager.js:130]: This is temporary only for `getQuotes` only. Like I said in item No. 2 after we merge the entire process we need to make support only to the quotes that our FE supported before the API i.e. celer and nxtp 😄.

If we in the first step of the migration allow our users to choose any other bridge than the aforementioned we need to make support in our FE using their SDK, like we've done creating `utils/nxtp.js` and `utils/cbridge.js`

File: src/client/js/utils/txBridgeManager.js (hunk 3 above)
Comment (yuripramos) [reply src/client/js/utils/txBridgeManager.js:229]: Yes. this PR is for getQuotes only

File: src/client/js/components/partials/bridge/AvailableRoutes.jsx [hunk 7]
```diff
@@ -115,14 +87,10 @@ export default function AvailableRoutes(props) {
...
-      {routes.length > 0 &&
-        _.map(routes, (item, i) => (
-          <RouteItemWrapper
-            handleChange={props.handleChange}
-            key={i}
-            data={item}
-            index={i}
-          />
+      {routes
+        ?.filter((item) => item.bridgeType === 'nxtp' || item.bridgeType === 'celer')
```
[the remark this answers is not in the document]
Comment (chillvybz) [reply src/client/js/components/partials/bridge/AvailableRoutes.jsx:91]: ??

File: src/client/js/components/partials/bridge/BridgeWidget.jsx (hunk 6 above)
Comment (chillvybz) [reply src/client/js/components/partials/bridge/BridgeWidget.jsx:32]: ??

File: src/client/js/utils/requests/utils.js (hunk 5 above)
Comment (chillvybz) [reply src/client/js/utils/requests/utils.js:4]: ??

File: src/client/js/utils/cbridge.js (hunk 4 above)
Comment (chillvybz) [reply src/client/js/utils/cbridge.js:14]: ??

**Pull request closed without merging**